src/Components/University/StuentDetails.tsx

```diff
@@ -379,6 +393,15 @@ export const StudentDetails: React.FC<Props> = (props: Props) => {
               </Card>
             </article>
           ))}
+          {activityData.length > 0 && (
```

Comment:
I think we do not need this check. It is handled inside the Pagination component.